packages/typegpu/src/data/index.ts

```diff
@@ -107,6 +107,25 @@ export {
 } from './dataTypes';
 export { PUBLIC_sizeOf as sizeOf } from './sizeOf';
 export { PUBLIC_alignmentOf as alignmentOf } from './alignmentOf';
+export {
+  builtin,
+  BuiltinVertexIndex,
+  BuiltinInstanceIndex,
+  BuiltinPosition,
+  BuiltinClipDistances,
+  BuiltinFrontFacing,
+  BuiltinFragDepth,
+  BuiltinSampleIndex,
+  BuiltinSampleMask,
+  BuiltinFragment,
+  BuiltinLocalInvocationId,
+  BuiltinLocalInvocationIndex,
+  BuiltinGlobalInvocationId,
+  BuiltinWorkgroupId,
+  BuiltinNumWorkgroups,
+  AnyBuiltin,
+  OmitBuiltins,
```

Comment:
Seems more like an internal type, lets omit it for now. (haha, pun not intended)
```suggestion
```